integration/subxt-tests/Cargo.toml

```diff
@@ -0,0 +1,28 @@
+[package]
+edition = "2021"
+name = "subxt-tests"
+version = "0.1.0"
+
+# See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
```

Comment:
Not needed, we know where to find it :grin:

Reply:
It's from cargo new, Aka nanny compiler.